internal/datastore/crdb/migrations/manager.go

```diff
@@ -1,6 +1,12 @@
 package migrations
 
-import "github.com/authzed/spicedb/pkg/migrate"
+import (
+	"github.com/authzed/spicedb/pkg/migrate"
+
+	"github.com/jackc/pgx/v4"
+)
+
+var noNonatomicMigration migrate.MigrationFunc[*pgx.Conn]
```

Comment:
Maybe `emptyMigration` or `noMigration`? The current name is more descriptive but is harder to parse

Reply:
I'm using typed nil to make sure it's clear at the call site which type of migration is being skipped.

Reply:
Yep, I'm just commenting on the name